src/openrct2/ride/Track.h

```diff
@@ -18,7 +18,9 @@
 
 constexpr const uint32_t RideConstructionSpecialPieceSelected = 0x10000;
 
-constexpr const int32_t BLOCK_BRAKE_BASE_SPEED = 0x20364;
+constexpr const uint8_t RCT2DefaultBlockBrakeSpeed = 2;
+constexpr const int32_t BlockBrakeBaseSpeed = 0x20364;
+constexpr const int32_t BlockBrakeSpeedOffset = BlockBrakeBaseSpeed - (RCT2DefaultBlockBrakeSpeed << 16);
```

Comment:
could you add a leading k to these constants `kBlockBrakeBaseSpeed` its slightly confusing with the new `BlockBrakeSpeed` member variable.